include/openssl/ssl3.h

```diff
@@ -25,6 +25,9 @@
 extern "C" {
 #endif
 
+/* Flag used on OpenSSL ciphersuite ids to indicate they are for SSLv3+ */
+# define SSL3_CK_CIPHERSUITE_FLAG                0x03000000
```

Comment:
You misunderstood me.  Take the flag off every ciphersuite, and put the new "v2" flag on those ciphers that are only sslv2.